docs/content/1.guide/1.configuration.md

```diff
@@ -4,9 +4,7 @@ icon: ri:settings-3-line
 
 # Configuration
 
-You can customize your Nitro server with single configuration file: `nitro.config.ts`.
-
-All deployment providers are built on the same options API.
+You can customize your Nitro server with single configuration file called `nitro.config.ts`. This file will be used by [deployement presets](/deploy) so you do not need to worry about them when configuring your server.
```

Comment:
THe left senstence does not means that -- it meant that presets are **made with** same interface 😅 Let's remove it altogether for now seems more confusing

```suggestion
You can customize your Nitro server with a single configuration file `nitro.config.ts`.
```